PP_APP/server/frontend/templates/rate.html

```diff
@@ -3,49 +3,75 @@
    </div>
-   <div class="row">
-      <div class="col-sm">
-         <div class="card mb-5" style="border-color:transparent">
-            <div class="card-body">
-               <h2 class="card-title">Step 1.</h2>
-               <p class="card-text">Take a picture from the TOP of your package.</p>
-            </div>
-            <img class="card-img-top" src="{% static 'img/sample_top.png' %}" height="250">
+   <form id="imageUploadForm" method="post" enctype="multipart/form-data">
+      <a id = "whats_this_id" class="text-black-100 black-par" data-toggle="collapse" href="#whats_this" role="button" aria-expanded="false"
+         aria-controls="whats_this">
+         + How does it work?
+      </a>
+      <div id="whats_this" class="collapse">
+         <div class="custom_card card-body">
+            <p>
+               Online retailers in particular are notorious for sending your items wrapped in wasteful packaging. How can we
+               get them to stop? By <strong>pressuring</strong> them of course!
+            </p>
+            <p>
+               But how can <strong>I</strong> help you ask? Grab a hold of a bad packaging you have received and follow the steps below.
+            </p>
+            <p>
+               We will use state of the art Computer Vision and Machine Learning techniques to give a rating to your package.
+            </p>
          </div>
-      </div>
-      <div class="col-sm">
-         <div class="card mb-5" style="border-color:transparent">
-            <div class="card-body">
-               <h2 class="card-title">Step 2.</h2>
-               <p class="card-text">Take a picture of the LOGO of your package.</p>
+         <h4 class="align-items-center text-center">&nbsp;</h4>
+         <div class="row">
+            <div class="col-sm">
+               <div class="card mb-5" style="border-color:transparent">
+                  <div class="card-body">
+                     <h2 class="card-title">Step 1.</h2>
+                     <p class="card-text">Take a picture from the TOP of your package.</p>
+                  </div>
+                  <img class="card-img-top" src="{% static 'img/sample_top.png' %}" height="250">
+               </div>
             </div>
-            <img class="card-img-top" src="{% static 'img/sample_side.png' %}" height="250">
-         </div>
-      </div>
-      <div class="col-sm">
-         <div class="card mb-5" style="border-color:transparent">
-            <div class="card-body">
-               <h2 class="card-title">Step 3.</h2>
-               <p class="card-text">Click submit and you will receive your result and score!</p>
+            <div class="col-sm">
+               <div class="card mb-5" style="border-color:transparent">
+                  <div class="card-body">
+                     <h2 class="card-title">Step 2.</h2>
+                     <p class="card-text">Take a picture of the LOGO of your package.</p>
```

Comment:
Take a picture of the LOGO ~~of~~ **on** your package.

Reply:
"Take a picture of the logo or the company name on your package"

In case there's no logo we use OCR to find the retailer.